src/sentinel.rs

```diff
@@ -209,7 +209,7 @@ impl<'a, Request, Name>
                 // We use raw bytes from the key here because in the current
                 // version of sodiumdioxide library PublicKey doesn't derive
                 // from Eq nor PartialEq. Once a version greater or equal to
-                // 0.0.5 is used, we can get rid of this unwrapping and then
+                // 0.0.6 is used, we can get rid of this unwrapping and then
```

Comment:
So what name of the subsection of the Todo section would you suggest? (Also should be OK'd by @Viv-Rajkumar I guess to keep some standard).

A second option would be if we just mark the comment as FIXME and whoever stumbles at the code can check it whether it's been added into the library?

And a third option would be to add the TODO into the Cargo.toml right where version of sodiumdioxide is specified. Just found out it should be fixed to 0.0.5 as the new version probably won't have asymmetrixbox.

I don't mind either way.